src/server/zset_family.h

```diff
@@ -97,6 +97,26 @@ class ZSetFamily {
   static void GeoAdd(CmdArgList args, ConnectionContext* cntx);
   static void GeoHash(CmdArgList args, ConnectionContext* cntx);
   static void GeoPos(CmdArgList args, ConnectionContext* cntx);
+  static void GeoDist(CmdArgList args, ConnectionContext* cntx);
 };
 
 }  // namespace dfly
+
+namespace {
```

Comment:
Anonymous namespaces shouldn't be in header files.
Please cut this code into the `.cc`, it's essentially implementation details.